tests/datasource/fluent/test_spark_google_cloud_storage_datasource.py

```diff
@@ -311,96 +223,16 @@ def test_csv_asset_with_non_string_batching_regex_named_parameters(
     mock_list_keys.return_value = object_keys
     asset = spark_gcs_datasource.add_csv_asset(
         name="csv_asset",
-        batching_regex=r"(.+)_(.+)_(?P<price>\d{4})\.csv",
     )
+    batching_regex = re.compile(r"(.+)_(.+)_(?P<price>\d{4})\.csv")
     with pytest.raises(ge_exceptions.InvalidBatchRequestError):
         # price is an int which will raise an error
-        asset.build_batch_request({"name": "alex", "timestamp": "1234567890", "price": 1300})
-
-
-@pytest.mark.big
-@pytest.mark.xfail(
-    reason="Accessing objects on google.cloud.storage using Spark is not working, due to local credentials issues (this test is conducted using Jupyter notebook manually)."  # noqa: E501
-)
-def test_get_batch_list_from_fully_specified_batch_request(
-    monkeypatch: pytest.MonkeyPatch,
-    spark_gcs_datasource: SparkGoogleCloudStorageDatasource,
-):
-    gcs_client: google.Client = cast(google.Client, MockGCSClient())
-
-    def instantiate_gcs_client_spy(self) -> None:
-        self._gcs = gcs_client
-
-    monkeypatch.setattr(
-        great_expectations.execution_engine.sparkdf_execution_engine.SparkDFExecutionEngine,
-        "_instantiate_s3_client",
-        instantiate_gcs_client_spy,
-        raising=True,
-    )
-    asset_specified_metadata = {"asset_level_metadata": "my_metadata"}
-    asset = spark_gcs_datasource.add_csv_asset(
-        name="csv_asset",
-        batching_regex=r"(?P<name>.+)_(?P<timestamp>.+)_(?P<price>\d{4})\.csv",
-        batch_metadata=asset_specified_metadata,
-    )
-
-    request = asset.build_batch_request({"name": "alex", "timestamp": "20200819", "price": "1300"})
-    batches = asset.get_batch_list_from_batch_request(request)
-    assert len(batches) == 1
-    batch = batches[0]
-    assert batch.batch_request.datasource_name == spark_gcs_datasource.name
-    assert batch.batch_request.data_asset_name == asset.name
-    assert batch.batch_request.options == {
-        "path": "alex_20200819_1300.csv",
-        "name": "alex",
-        "timestamp": "20200819",
-        "price": "1300",
-    }
-    assert batch.metadata == {
-        "path": "alex_20200819_1300.csv",
-        "name": "alex",
-        "timestamp": "20200819",
-        "price": "1300",
-        **asset_specified_metadata,
-    }
-    assert batch.id == "spark_gcs_datasource-csv_asset-name_alex-timestamp_20200819-price_1300"
-
-    request = asset.build_batch_request({"name": "alex"})
-    batches = asset.get_batch_list_from_batch_request(request)
-    assert len(batches) == 2
```

Comment:
this test fails because the `SparkExecutionEngine` doesn't have the attribute that we're trying to patch:
```
>       monkeypatch.setattr(
            great_expectations.execution_engine.sparkdf_execution_engine.SparkDFExecutionEngine,
            "_instantiate_s3_client",
            instantiate_gcs_client_spy,
            raising=True,
        )
E       AttributeError: <class 'great_expectations.execution_engine.sparkdf_execution_engine.SparkDFExecutionEngine'> has no attribute '_instantiate_s3_client'
```
As far as I can tell this test was never completed.

Reply:
the execution engine also doesn't have `_instantiate_gcs_client`, which would make more sense than `_isntantiate_s3_client` as written.